### PR TITLE
WIP : Cbp 7139 adding tarflag

### PR DESCRIPTION
**Pull request by AmitRamachandrann**

### CODE REVIEW

File: .cloudbees/testing/action.yml [hunk 1]
```diff
@@ -84,6 +84,10 @@ inputs:
   gitlab-server-url:
     description: The base URL for the GitLab instance that you are trying to clone from, will use environment defaults (i.e. the GITLAB_SERVER_URL environment variable) to fetch from the same instance that the workflow is running from unless specified. Example URLs are https://gitlab.com or https://my-gl-server.example.com
     required: false
+  create-tarfile:
```
Comment (stephenc) [inline .cloudbees/testing/action.yml:87]: this needs to be kept in sync with the root action.yaml. Why am I not seeing the matching change there?

Comment (AmitRamachandrann) [reply .cloudbees/testing/action.yml:87]: I thought it should be updated since I added a new flag for the checkout action.

Comment (stephenc) [reply .cloudbees/testing/action.yml:87]: you have to add the flag in both places

**Comment by AmitRamachandrann** [comment]
Might take a better modular approach for creating tar file while checkout

**Pull request closed without merging**